scripts/generate_gff3_db.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
Ugh. Are GFF files that awful? It seems a cop out to expect the repo manager write their own parser for each GFF file. Anyway, ignore the above comment for now, we should discuss this after the initial drop into the sequence annotations branch.